### ISSUE TITLE
[Component] [collapse-item] el-collapse-item renders styles for a focused state

### ISSUE THREAD

**Issue by xushangning** [element-plus/element-plus#9393]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.37`
- Element Plus Version: `2.2.14`
- Browser / OS: `Chrome 104.0.5112.79, macOS 12.5.1`
- Build Tool: `CDN`

### Related Component

- `el-collapse-item`

### Reproduction Link

[Docs](https://element-plus.org/en-US/component/collapse.html)

### Steps to reproduce

We use the el-collapse component under the subsection "Basic usage" for reproducing the bug.

1. Expand and collapse the the item with name "Feedback."
2. Click any area OUTSIDE THE BROWSER so that your browser window loses focus. Then, click inside the browser anywhere but OUTSIDE THE COMPONENT. Repeat twice.

### What is Expected?

Nothing happens.

### What is actually happening?

The "Feedback" item turns blue.

### Additional comments

This behavior is also reproducible in our own project with Element Plus 2.2.6, Webpack build.

It is an unexpected and undocumented behavior of the collapse component, likely caused by the collapse item coming into focus. The Chrome debugger shows that the focus event is handled at
https://github.com/element-plus/element-plus/blob/ea17f71e2169462aa3461a4993c4baa8b7e03c59/packages/components/collapse/src/collapse-item.vue#L79-L87

A screencast is also attached below that shows the bug itself and how it is reproduced.

https://user-images.githubusercontent.com/32517059/185680829-ba98b81e-2e13-412d-af87-9cd59433c88c.mov

**Comment by ryuhangyeong** [comment]
Is the method I tested correct? It cannot be reproduced.

https://user-images.githubusercontent.com/27342882/185728201-636e564f-1eb7-4442-862f-9e5930949636.mp4

**Comment by xushangning** [comment]
> Is the method I tested correct? It cannot be reproduced.
> 
> It seems impossible to test in my environment (window).
> 
>  GOMCAM.20220820_1301380186.mp4

I think you should use the first collapse component example in the page, the one under the subsection "Basic usage" and not in the accordion mode.

**Comment by ryuhangyeong** [comment]
https://user-images.githubusercontent.com/27342882/185734260-a84588dc-8e9a-4bcf-8e52-bd5dec7337e4.mp4

Same here

**Comment by xushangning** [comment]
> GOMCAM.20220820_1629150432.mp4 
> 
> Same here, I'll check it out in the Mac environment as well.

That's weird. Your repro steps are definitely correct. Just tested on another MacBook and it can still be reliably reproduced on both Chrome and Safari.

I won't have my Windows computer until several days later. What's your Windows version and browser?

**Comment by ryuhangyeong** [comment]
IE11, Chrome 104.0.5112.82

**Comment by xushangning** [comment]
Not reproducible on Edge 104.0.1293.63 or Chrome 104.0.5112.102. Still reproducible on macOS though.

**Comment by ryuhangyeong** [comment]
Yes, I have also seen this issue on Mac OS. But now my development environment is windows, so it's difficult to edit.

**Comment by xushangning** [comment]
> But now my development environment is windows, so it's difficult to edit.

That's alright for me as the issue is not very disturbing. Just fix at your convenience.

**Comment by Dsaquel** [comment]
Can't reproduce, maybe with the time being it is now resolved. 

If the issue persist feel free to reopen.
